.jenkins

```diff
@@ -36,7 +36,7 @@ pipeline {
                         sh 'ccache --zero-stats'
                         sh '''rm -rf build && mkdir -p build && cd build && \
                               cmake \
-                                -DCMAKE_BUILD_TYPE=Release \
+                                -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
??

Reply:
Fixed in [`3d32df0` (#4138)](https://github.com/kokkos/kokkos/pull/4138/commits/3d32df071803f492a4c0c8fc460ec92f0d7ca6e0).